examples/Schelling/server.py

```diff
@@ -1,6 +1,35 @@
-from Schelling import SchellingModel
 from mesa.visualization.ModularVisualization import ModularServer
 from mesa.visualization.modules import CanvasGrid, ChartModule, TextElement
+from mesa.visualization.TextVisualization import (TextData, TextGrid,
+    TextVisualization)
+
+from model.world import SchellingModel
```

Comment:
This line needs to be changed to `from schelling import SchellingModel` OR we need to rename `schelling.py` to `model.py`. I believe I would prefer `model.py` rename. Thoughts? @dmasad